shell/app/modules/project/types/project.d.ts

```diff
@@ -21,6 +21,11 @@ declare namespace PROJECT {
     TEST: T
   }
 
+  interface ProjectListTypeParams {
```

Comment:
If the interface is used only in a single file, then I think there is no need to define it in global namespace. U can just define it in the specific file